hicexplorer/hicFindTADs.py

```diff
@@ -900,13 +781,18 @@ def compute_spectra_matrix(args, matrix=None):
         hic_ma = matrix
     else:
         hic_ma = hm.hiCMatrix(args.matrix)
+
     # remove self counts
     hic_ma.diagflat(value=0)
 
     # mask bins without any information
     hic_ma.maskBins(hic_ma.nan_bins)
     orig_intervals = hic_ma.cut_intervals
 
+    # use zscore matrix
+    sys.stderr.write("Computing z-score matrix...\n")
```

Comment:
This is not standardized in the code but in general, I am trying to output messages using standard error. Standard ouput is reserved for output that may be passed to other programs.